tensorhive/controllers/restriction/update_restriction_controller.py

```diff
@@ -0,0 +1,41 @@
+from tensorhive.authorization import admin_required
+from tensorhive.models.Restriction import Restriction
+from sqlalchemy.orm.exc import NoResultFound
+from tensorhive.config import API
+import logging
+log = logging.getLogger(__name__)
+R = API.RESPONSES['restriction']
+G = API.RESPONSES['general']
+
+
+@admin_required
+def update(id, newValues):
+    new_values = newValues
```

Comment:
That seems unnecessary... any particular reason why we wouldn't like to use this_notation in the method's argument straight away?